e2e-tests/src/types/parameter.d.ts

```diff
@@ -1,9 +1,17 @@
 type EffectiveArguments = {
-  arguments: ArgumentsMap;
-  errors: ParametersErrorMap;
+  arguments?: ArgumentsMap;
```

Comment:
Could you help me understand why these properties are now optional? My brain function is too low right now to try to figure it out myself 🥴

Reply:
😆 This particular change just updates the existing type within our e2e tests to match what `getActivityEffectiveArguments` already returns: optional `errors` and `arguments` (i.e. one or the other).

I then matched this same behavior for `getActivityEffectiveArgumentsBulk` to return a list of these same objects with optional fields.